server/utils/azureAppInsights.ts

```diff
@@ -15,18 +14,16 @@ function version(): string {
 }
 
 export function initialiseAppInsights(): void {
-  // Loads .env file contents into | process.env
-  config()
```

Comment:
Not needed. `.env` config is loaded on application start via ` -r dotenv/config`. Updated `package.json` so this is also true of local dev builds.